clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/AvroGenericDaVinciClient.java

```diff
@@ -659,14 +659,9 @@ private void initBackend(
                 backend.close();
               }
             });
-      } else if (VeniceSystemStoreType
-          .getSystemStoreType(clientConfig.getStoreName()) != VeniceSystemStoreType.META_STORE) {
-        logger.info("Da Vinci Backend exists, reusing existing backend for client: " + clientConfig.getStoreName());
-        // Do not increment DaVinciBackend reference count for meta system store da-vinci clients. Once the last user
-        // da-vinci
-        // client is released the backend can be safely deleted since meta system stores are meaningless without user
-        // stores,
-        // and they are cheap to re-bootstrap.
+      } else if (!VeniceSystemStoreType.META_STORE.isSystemStore(clientConfig.getStoreName())) {
+        // Only increment reference count for meta system store if it's used for fast client. Otherwise, close
```

Comment:
Is `fast client` related comment still relevant?

Reply:
Good catch, removed.